packages/neo-one-website/src/components/docs/Sidebar.tsx

```diff
@@ -0,0 +1,82 @@
+// tslint:disable strict-boolean-expressions
+import * as React from 'react';
+import { MdArrowDropDown } from 'react-icons/md';
+import { Link as RouterLink } from 'react-static';
+import { Box, Button, Grid, Hidden, styled, Toolbar } from 'reakit';
+import { prop } from 'styled-tools';
+// import { Link, Logo } from '../elements';
+// import { ComponentProps } from '../types';
+
+const StyledBox = styled(Box)`
+  background-color: ${prop('theme.gray1')};
+`;
+
+const StyledToolbar = styled(Toolbar)`
+  background-color: ${prop('theme.gray2')};
+`;
+
+const Page = styled(Toolbar.Content)`
+  white-space: nowrap;
+`;
+
+const Section = styled(Button)`
+  border-bottom: 3px solid transparent;
+  border-color: ${prop('theme.accent')};
+`;
+
+const NavigationLink = styled(RouterLink)`
+  display: flex;
+  align-items: center;
+  font-size: 12px;
+  height: 100%;
+  width: 100%;
+  padding-top: 5px;
+  color: ${prop('theme.black')};
+  text-decoration: none;
+
+  &:hover {
+    border-color: ${prop('theme.accent')};
+    color: ${prop('theme.accent')};
+  }
+
+  &.active {
+    border-color: ${prop('theme.accent')};
+  }
+`;
+
+export const Sidebar = () => (
+  <StyledBox>
+    <StyledToolbar>
+      <Hidden.Container>
+        {({ visible, toggle }) => (
+          <Page>
+            <Grid>
+              <Section onClick={toggle}>
+                Installation
+                <MdArrowDropDown />
+              </Section>
+              {visible && (
+                <Grid>
+                  <Toolbar.Focusable
+                    data-test="docs-getting-started-sidebar"
+                    as={NavigationLink}
+                    to="/docs/getting-started"
+                  >
+                    Getting Started
```

Comment:
This should be generated from the docs in the folders - not statically specified here.